app/components/workbench/components/CommitEditor.tsx

```diff
@@ -43,6 +44,10 @@ export const CommitEditorComponent: React.FunctionComponent<CommitEditorProps> =
 
   const { username, name } = qriRef
 
+  React.useEffect(() => {
+    ReactTooltip.rebuild()
+  }, [])
```

Comment:
Unless this function _must_ be bound to the current context, this should work and be a little cleaner:
```suggestion
  React.useEffect(ReactTooltip.rebuild, [])
```

While we're here can we get a basic comment on what this does? I'm guessing tooltips need to be rebuilt on render, but is there a specific reason? data-model specific stuff in tooltips?

Reply:
We build the tooltips in the App component. Anything that is not rendered in the first render cycle won't be binded to the ReactTooltip component. So on first render of the component, we rebuild the tooltips.

I'll add the comment!